### PR TITLE
Updated Quick and Nimble

### PR DESCRIPTION
**Pull request by Widcket**
### Changes

This PR updates the test dependencies [Quick](https://github.com/Quick/Quick/releases/tag/v3.0.0) and [Nimble](https://github.com/Quick/Nimble/releases/tag/v9.0.0) to the latest majors, making it possible to test JWTDecode.swift with Xcode 12.

**The new majors of Quick and Nimble no longer support Swift versions < 5.2. This means that the _tests_ can no longer be run on Xcode versions < 11.4. The CircleCI config has been updated accordingly.**

Also a notice has been added to the README warning developers using Carthage, as they will need to use only Xcode 11.4+ from the next minor onwards.

### References

Same changes in Auth0.swift: https://github.com/auth0/Auth0.swift/pull/421
Same changes in SimpleKeychain: https://github.com/auth0/SimpleKeychain/pull/97

### Testing

* [ ] This change adds unit test coverage
* [X] This change has been tested on the latest version of the platform/language or why not

### Checklist

* [X] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
* [X] All existing and new tests complete without errors
* [X] All active GitHub checks have passed

### CODE REVIEW

**Comment by Widcket** [comment]
@lbalmaceda can you please update the branch protection settings? The only CircleCI jobs that should be marked as required are **build-and-test-iOS-swift-5_2**, **build-and-test-macOS-swift-5_2**, **build-and-test-tvOS-swift-5_2** as the others have been removed. 
Also Codecov is detecting a coverage change, despite that no code was touched. The coverage reports are generated by Xcode, and it was updated to version 11.7. That might be the source of the reported code coverage difference.

**Comment by cocojoe** [comment]
> @lbalmaceda can you please update the branch protection settings? The only CircleCI jobs that should be marked as required are **build-and-test-iOS-swift-5_2**, **build-and-test-macOS-swift-5_2**, **build-and-test-tvOS-swift-5_2** as the others have been removed.
> Also Codecov is detecting a coverage change, despite that no code was touched. The coverage reports are generated by Xcode, and it was updated to version 11.7. That might be the source of the reported code coverage difference.

Done

File: README.md [hunk 1]
```diff
@@ -6,13 +6,16 @@
...
-![Swift 5.1](https://img.shields.io/badge/Swift-5.1-orange.svg?style=flat-square)
+![Swift 5.3](https://img.shields.io/badge/Swift-5.3-orange.svg?style=flat-square)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2FJWTDecode.swift.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2FJWTDecode.swift?ref=badge_shield)
 
 This library will help you check [JWT](https://jwt.io/) payload
 
 > This library doesn't validate the token, any well formed JWT can be decoded from Base64Url.
 
+## Important Notices
+- [The latest majors of Quick and Nimble](https://github.com/auth0/JWTDecode.swift/pull/114) require that developers using Carthage **must** use Xcode 11.4+ only, starting from JWTDecode.swift [2.5.0](https://github.com/auth0/JWTDecode.swift/releases/tag/2.5.0).
```
Comment (cocojoe) [inline README.md:17]: I would update requirements to match 11.4+, give we're not testing on anything less now, can't say for sure that it works.

**Review by cocojoe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Widcket) [reply README.md]: I removed the notice and updated the Xcode requirements to 11.4+

Comment (MagFer) [reply README.md]: I'm pointing to this PR in order to use JWTDecode.swift in a Xcode 12 project, is going to be merged 🙂?

**Review by cocojoe — APPROVED** [review]

**Pull request merged**